subworkflows/local/annotate_genomes.nf

```diff
@@ -27,7 +27,7 @@ workflow ANNOTATE_GENOMES {
     abricate_report = channel.empty()
 
     if(!params.skip_bakta && params.bakta.db){
-        db_file = Channel.value("${params.bakta.db}")
+        db_file = Channel.value("${params.bakta.db}") // Hows is this working without being marked as a file...
```

Comment:
As mentioned earlier, it's probably just passing it as something like a string, and the file is being read from where it original was with no staging.

Reply:
Removed the comment, and created an issue here: https://github.com/phac-nml/mikrokondo/issues/97